programs/pkey/pk_decrypt.c

```diff
@@ -78,32 +78,35 @@ int main( int argc, char *argv[] )
         polarssl_printf( "usage: pk_decrypt <key_file>\n" );
 
 #if defined(_WIN32)
-        polarssl_printf( "\n" );
+        polarssl_printf( "\n  + Press Enter to exit this program.\n" );
+        fflush( stdout ); getchar();
```

Comment:
These changes are not in the original backport. While is not a crucial difference, it would be good to try to keep both versions consistent to ease backports of future changes.

Reply:
Perhaps it would be better to just move the init statements above this and put a goto here to make development and backport a bit more consistent? Please note taht this is just to ease maintenance in the future